apps/admin/backend/src/cvr_files.ts

```diff
@@ -665,7 +665,7 @@ export async function addCastVoteRecordReport({
               CVR_BALLOT_IMAGES_SUBDIRECTORY,
               CVR_BALLOT_LAYOUTS_SUBDIRECTORY
             )
-            .replace(/jpg|jpeg/, 'layout.json');
+            .replace(/(jpg|jpeg)$/, 'layout.json');
```

Comment:
I think the normalized filenames with the rust interpreter are different than they were before, requiring me to be more precise here. The new filenames are like: `324123-12312-uuid-1234123.jpg-12312-3129-1232-uuid-12321-normalized.jpg`

Reply:
Kind of a weird filename format, but hardening the regex seems like a net positive anyway